ui/pages/swaps/selected-token/selected-token.js

```diff
@@ -0,0 +1,69 @@
+import React, { useContext } from 'react';
```

Comment:
Most of the code here is reused from this component: https://github.com/MetaMask/metamask-extension/blob/develop/ui/pages/swaps/dropdown-search-list/dropdown-search-list.js